platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -148,6 +148,14 @@ The [TestNGOptions](javadoc/org/gradle/api/tasks/testing/testng/TestNGOptions.ht
 
 More information about this option is available in the [TestNG documentation](https://testng.org/#_command_line_parameters).
 
+### Adoption of the Problems API with Java compilation
+
+The Java compilation infrastructure has been updated to use the [Problems API](https://docs.gradle.org/8.6/userguide/implementing_gradle_plugins.html#reporting_problems).
+This change will supply the IDEs and Tooling API clients with structured, rich information about compilation issues.
```

Comment:
```suggestion
This change will supply the Tooling API clients with structured, rich information about compilation issues.
```

Reply:
Do you think it's not worth pointing out?

Reply:
IDEs are tooling api clients, that's all.